content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -20,9 +20,9 @@ cluster. This can help to reduce costs and improve performance.
 
 ## Using Service Internal Traffic Policy
 
-Once you have enabled the `ServiceInternalTrafficPolicy`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/),
-you can enable an internal-only traffic policy for a
+Using beta `ServiceInternalTrafficPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is already enabled by default,
+Only when the feature gate is enabled you can enable an internal-only traffic policy for a
```

Comment:
This can be simply put as the following?

```
When the `ServiceInternalTrafficPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
is enabled, you can enable an internal-only traffic policy for a ...
```

Reply:
> This can be simply put as the following?
> 
> ```
> When the `ServiceInternalTrafficPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
> is enabled, you can enable the internal-only traffic policy for a ...
> ```

But @tengqm if we set the documentation like this then reader will not know the by default property of beta version here, do you think ? It will be okay if we don't mention the by default behavior of beta version.

Reply:
I don't think we need to emphasize that this gate is in beta and enabled by default. If necessary, we can add another sentence before this one, e.g.

```
The `ServiceInternalTrafficPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
is a Beta feature and enabled by default.
When the feature is enabled, you can enable the internal-only traffic policy for a ...
```

Does this make senses?

Reply:
yup it makes sence.